homeassistant/components/microbees/button.py

```diff
@@ -0,0 +1,54 @@
+"""Button integration microBees."""
+from typing import Any
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesActuatorEntity
+
+BUTTON_TRANSLATIONS = {51: "button_gate", 91: "button_panic"}
+BUTTON_PRODUCT_IDS = {51, 91}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the microBees button platform."""
+    coordinator: MicroBeesUpdateCoordinator = hass.data[DOMAIN][
+        entry.entry_id
+    ].coordinator
+    async_add_entities(
+        MBButton(coordinator, bee_id, button.id)
+        for bee_id, bee in coordinator.data.bees.items()
+        if bee.productID in BUTTON_PRODUCT_IDS
```

Comment:
```suggestion
        if bee.productID in BUTTON_TRANSLATIONS.keys()
```

This way you could avoid create a second dict and maintaining the IDs twice. Maybe use `BUTTON_PRODUCT_IDS` as the const name instead